ntbs-service/DataMigration/ImportValidator.cs

```diff
@@ -95,6 +97,42 @@ public ImportValidator(IImportLogger logger, IReferenceDataRepository referenceD
             }
         }
 
+        private static void TrimAndCleanLongFormNotes(Notification notification)
+        {
+            notification.ClinicalDetails.Notes = TrimAndCleanString(notification.ClinicalDetails.Notes);
+
+            foreach (var address in notification.SocialContextAddresses)
+            {
+                address.Details = TrimAndCleanString(address.Details);
+            }
+            foreach (var venue in notification.SocialContextVenues)
+            {
+                venue.Details = TrimAndCleanString(venue.Details);
+            }
+            foreach (var treatmentEvent in notification.TreatmentEvents)
+            {
+                treatmentEvent.Note = TrimAndCleanString(treatmentEvent.Note);
+            }
+            foreach (var exposure in notification.MBovisDetails.MBovisAnimalExposures)
+            {
+                exposure.OtherDetails = TrimAndCleanString(exposure.OtherDetails);
+            }
+            foreach (var exposure in notification.MBovisDetails.MBovisExposureToKnownCases)
+            {
+                exposure.OtherDetails = TrimAndCleanString(exposure.OtherDetails);
+            }
+            foreach (var exposure in notification.MBovisDetails.MBovisOccupationExposures)
+            {
+                exposure.OtherDetails = TrimAndCleanString(exposure.OtherDetails);
+            }
+            foreach (var exposure in notification.MBovisDetails.MBovisUnpasteurisedMilkConsumptions)
+            {
+                exposure.OtherDetails = TrimAndCleanString(exposure.OtherDetails);
+            }
```

Comment:
I've also added in the removal of tabs on import to this PR. I was expecting to do this in migration but decided that this might be better place for it. The trimming and removing of tabs is quite neat in C#, but applying it to all of the different fields is a bit ugly. Can you think of a tidier way of writing this? It feels quite repetitive but I haven't thought of a good way of DRY'ing it out